db/db_tunables.h

```diff
@@ -1664,13 +1664,19 @@ REGISTER_TUNABLE("dohast_verbose",
 REGISTER_TUNABLE("dohsql_max_queued_kb_highwm",
                  "Maximum shard queue size, in KB; shard sqlite will pause "
                  "once queued bytes limit is reached.",
-                 TUNABLE_INTEGER, &gbl_dohsql_max_queued_kb_highwm, 0, NULL,
+                 TUNABLE_INTEGER, &gbl_dohsql_max_queued_kb_highwm, 10000, NULL,
```

Comment:
This is flags, not default (though this would be the perfect place to specify defaults...)